requirements.txt

```diff
@@ -91,14 +91,14 @@ tzlocal==1.3
 unicodecsv==0.14.1
 Unidecode==1.0.22
 urllib3==1.26.5
-watchdog==2.2.1
+watchdog==3.0.0
```

Comment:
Also seems suspicious..

Reply:
Seems ok to me: https://github.com/gorakhargosh/watchdog/blob/master/changelog.rst